### ISSUE TITLE
Template format for version 3.01

### ISSUE THREAD

**Issue by randym127** [cipchk/ngx-countdown#117]
Can I get the documentation for the template for 3.01?  What you can see there now is not very helpful.

**Comment by randym127** [comment]
Specifically can it display years?

**Comment by cipchk** [comment]
Of course, for the `3.x` version, you can solve the rendering problem with [repaint](https://github.com/cipchk/ngx-countdown/blob/3.0.1/src/app/tpl/flip/flip.component.ts#L13).
